src/main/kotlin/org/rust/cargo/runconfig/test/CargoTestEventsConverter.kt

```diff
@@ -198,20 +239,34 @@ class CargoTestEventsConverter(
         suitesStack.add(suite)
     }
 
+    private enum class State {
+        ReceivesStartMessage,
```

Comment:
`Receives` prefix looks redundant because all enum variants use the same prefix & doesn't help (at least for me) to understand enum meaning without usages reading. Maybe it's better to drop it and add short javadoc?